src/utils.zig

```diff
@@ -1907,6 +1907,206 @@ test "read/write mixed" {
 	try std.testing.expect(reader.remaining.len == 0);
 }
 
+pub fn SparseSet(comptime T: type, comptime IdType: type) type { // MARK: SparseSet
```

Comment:
Please assert that the `noValue` field has the maximum integer value of the backing enum of the IdType